CHANGED

```diff
@@ -1,3 +1,5 @@
+31.08.2019
+ 14_SD_WS_Maverick.pm: added support for device TFA 14.1504
```

Comment:
Ergänzungsvorschlag:
14_SD_WS_Maverick.pm: 
         feature: added support for device TFA 14.1504
         changed: Removed internals which are identical to readings
         changed: Removed declaration of stateformat
         changed: Optimized some code.

Reply:
Ist erledigt.